openmetadata-ui/src/main/resources/ui/src/components/DataQuality/IncidentManager/TestCaseResultTab/TestCaseResultTab.component.tsx

```diff
@@ -220,6 +220,7 @@ const TestCaseResultTab = () => {
               </Col>
               <Col span={24}>
                 <SchemaEditor
+                  className="query-editor-min-h-60"
```

Comment:
@ShaileshParmar11 , can we use this class `custom-query-editor`? it will handle the height as well.